contrastscan-remote/src/main/java/com/contrastsecurity/scan/PreScanDataGenerator.kt

```diff
@@ -0,0 +1,81 @@
+package com.contrastsecurity.scan
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.slf4j.LoggerFactory
+import java.io.FileWriter
+import java.io.IOException
+import java.io.Writer
+import java.net.URI
+import java.nio.file.*
+import java.nio.file.attribute.BasicFileAttributes
+
+sealed class PreScanDataGenerator {
+    companion object {
+
+        @JvmStatic
+        private val logger = LoggerFactory.getLogger(javaClass.enclosingClass)
+
+        @Throws(Exception::class)
```

Comment:
@Throws is not necessary while you are in the Kotlin universe. It adds metadata to the class files for interop with Java